pyproject.toml

```diff
@@ -96,6 +96,11 @@ warn_redundant_casts = true
 warn_unused_configs = true
 warn_unused_ignores = true
 
+# Ignore mypy errors for modules imported from datatree_.
+[[tool.mypy.overrides]]
```

Comment:
Should this subsection heading appear twice?

Reply:
Seems ok.  The double bracket is an [array of tables](https://toml.io/en/v1.0.0#array-of-tables) in toml land, and we already have multiple values in the file.